pkg/clusteragent/autoscaling/workload/model/pod_autoscaler.go

```diff
@@ -38,6 +38,9 @@ type PodAutoscalerInternal struct {
 	// name is the name of the PodAutoscaler
 	name string
 
+	// annotations are the annotations of the PodAutoscaler
+	annotations map[string]string
```

Comment:
Do we actually need all annotations, shouldn't we directly parse what we need into structured data?